examples/send_message.py

```diff
@@ -1,29 +1,31 @@
-"""Buy some ram to some account."""
-
-
-import eospyo
-
-data = {
-    "payer": "me.wam",
-    "receiver": "me.wam",
-    "quant": "5.00000000 WAX",
-}
-
-auth = eospyo.Authorization(actor="me.wam", permission="active")
-
-action = eospyo.Action(
-    account="eosio",
-    name="buyram",
-    data=data,
-    authorization=[auth],
-)
-
-raw_transaction = eospyo.Transaction(actions=[action])
-
-net = eospyo.WaxTestnet()
-linked_transaction = raw_transaction.link(net=net)
-
-key = "a_very_secret_key"
-signed_transaction = linked_transaction.sign(key=key)
-
-resp = signed_transaction.send()
+"""Send a message."""
+
+
+import eospyo
+
+
+data=[
+        eospyo.Data(name="from", value=eospyo.types.Name("me.wam")),
+        eospyo.Data(name="message", value=eospyo.types.String("hello from eospyo"),),

```

Comment:
we're missing the "quant" prop here, no?

Reply:
This is just a message action, so I don't believe the quant prop should be included, the reason I didn't include/convert examples with quant props is because the quant prop requires the asset type, which is not defined yet in the [types.py](https://github.com/FACINGS/eospyo/blob/main/eospyo/types.py) file, so currently any code/example that requires the asset type will not work.